.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint 
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - run: yarn install  
+      - name: lint
+        run: npm run lint
```

Comment:
Shouldn't this be yarn instead of npm?

Reply:
Yea I think you are right, I can change it to yarn.